android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -209,12 +206,14 @@ private void initializePlayer() {
             player.setMetadataOutput(this);
             exoPlayerView.setPlayer(player);
             audioBecomingNoisyReceiver.setListener(this);
-            setPlayWhenReady(!isPaused);
             playerNeedsSource = true;
 
             PlaybackParameters params = new PlaybackParameters(rate, 1f);
             player.setPlaybackParameters(params);
         }
+
+        setPlayWhenReady(!isPaused);
```

Comment:
Why was this change made, I'm not sure what it does. When testing with my changes, the video plays automatically even if I switch sources.